dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/AbstractJdbcTableManager.java

```diff
@@ -387,7 +393,7 @@ protected void executeSilently( String sql )
         }
         catch ( BadSqlGrammarException ex )
```

Comment:
In "master" we are, know, catching `DataAccessException`, should it be the case here too?